src/System.CommandLine.Tests/TestApps/NativeAOT/NativeAOT.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<!-- producing more detailed log output -->
+		<TrimmerSingleWarn>false</TrimmerSingleWarn>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<SystemCommandLineDllPath Condition="'$(SystemCommandLineDllPath)'==''">..\..\..\System.CommandLine\bin\Release\net6.0\System.CommandLine.dll</SystemCommandLineDllPath>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<Reference Include="SystemCommandLineDll">
+			<HintPath>$(SystemCommandLineDllPath)</HintPath>
+		</Reference>
+		<PackageReference Include="Microsoft.DotNet.ILCompiler" Version="7.0.0-*" />
```

Comment:
Can we instead hard-code a version, and then set up a Maestro update to keep it updated?